moveit_core/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ find_library(LIBFCL_LIBRARIES fcl HINTS ${LIBFCL_PC_LIBRARY_DIRS})
 find_package(octomap REQUIRED)
 find_package(urdfdom REQUIRED)
 find_package(urdfdom_headers REQUIRED)
+find_package(Bullet REQUIRED)
```

Comment:
Shouldn't we use bullet3_ros?

Reply:
That is the ultimate plan. This uses the same fork that Tesseract uses at the moment, just so Jens can get a checker up and comparable quickly.

Reply:
bullet3_ros is actually used, this is still there because initially I thought we can use the debian Bullet version. Will remove this.

Reply:
Whatever makes more sense. If the debian version works just fine and is broadly available I don't see an issue with that.